### PR TITLE
Fixes outdated Consular directives 

### PR DESCRIPTION
**Pull request by connorjg1**
Consular directives were outdated and mentioned NanoTrasen/Station as the primary megacorp/location. This fixes that by changing it to the SCC and Horizon respectively.

### CODE REVIEW

**Comment by connorjg1** [comment]
!review

**Review by mikomyazaki — CHANGES_REQUESTED** [review]
Would prefer if you could use the `current_map.name`, `current_map.boss_name` etc. Vars instead of writing the names in directly. Check the horizon map definition for these.

**Comment by connorjg1** [comment]
I'm sure not why I'm getting an error message for the Vaurca objectives since I can't see anything wrong with it, but I'll try and solve it

File: code/modules/background/citizenship/citizenship.dm [hunk 1]
```diff
@@ -10,14 +10,14 @@
 
 	switch(mission_level)
 		if(REPRESENTATIVE_MISSION_HIGH)
-			rep_objectives = pick("Collect evidence of NanoTrasen being unfair or oppressive against employees from [name], to be used as leverage in future diplomatic talks",
+			rep_objectives = pick("Collect evidence of the Stellar Corporate Conglomerate being unfair or oppressive against employees from [name], to be used as leverage in future diplomatic talks",
```
Comment (mikomyazaki) [inline code/modules/background/citizenship/citizenship.dm:13]: Missed a few.

**Review by mikomyazaki — CHANGES_REQUESTED** [review]

**Review by JohnWildkins — APPROVED** [review]
address myazaki's review and fix conflicts and this is fine

File: code/modules/background/citizenship/citizenship.dm (hunk 1 above)
Comment (connorjg1) [reply code/modules/background/citizenship/citizenship.dm]: Addressed. I think that got them all.

**Pull request merged**